x-pack/plugins/enterprise_search/public/applications/workplace_search/views/settings/components/oauth_application.test.tsx

```diff
@@ -96,25 +96,55 @@ describe('OauthApplication', () => {
     expect(wrapper.find(CredentialItem)).toHaveLength(2);
   });
 
-  it('renders license modal', () => {
-    setMockValues({
-      hasPlatinumLicense: false,
-      oauthApplication,
+  describe('non-platinum license content', () => {
+    beforeEach(() => {
+      setMockValues({
+        hasPlatinumLicense: false,
+        oauthApplication,
+      });
     });
-    const wrapper = shallow(<OauthApplication />);
-
-    expect(wrapper.find(EuiModal)).toHaveLength(1);
-  });
 
-  it('closes license modal', () => {
-    setMockValues({
-      hasPlatinumLicense: false,
-      oauthApplication,
+    it('renders pageTitle', () => {
+      const wrapper = shallow(<OauthApplication />);
+
+      expect(wrapper.prop('pageHeader').pageTitle).toMatchInlineSnapshot(`
```

Comment:
Question: Why did you opt into using a snapshot here? We're not using them anywhere else IIRC.

Reply:
Yeah, we're using it in several places now. It was first recommended by @constancecchen [here](https://github.com/elastic/kibana/pull/102826#discussion_r656785162). For this particular use-case, I was having trouble using `.dive` to get to the props for assertion and this was a fairly simple way to get coverage.

Reply:
We probably do want to be deliberate about when and where we use inline snapshots rather than falling back to them as a default, but this specific use case (making it easier/not having to dive into a nested prop) does make sense for a snapshot just IMO!